build.nix

```
@@ -89,17 +89,25 @@ let
         checkout=$(echo "$dep" | jq -cMr '.checkout')
         url=$(echo "$dep" | jq -cMr '.url')
         tomls=$(find $checkout -name Cargo.toml)
+        rev=$(echo "$dep" | jq -cMr '.rev')
+        echo "DEP IS $dep"
```

Maybe to help future debugging?
```suggestion
        echo "Dependency: $dep (rev: $rev from $url) checked out at $checkout"
```

---

wops! left over. thanks!